new-nextjs-client/src/entities/app-user.ts

```diff
@@ -16,7 +26,7 @@ class AppUser implements IAppUser {
   private _years: number[] = [];
   private _authToken: string | null = null;
   private _degreePlan = new DegreePlan();
-  private _courseBag: Course[] = [
+  private _courseBag: ICourse[] = [
```

Comment:
What is the `ICourse` array name stands for in `app-user`?

Reply:
Interface!